test/e2e/pod_infra_container_test.go

```diff
@@ -125,6 +125,19 @@ var _ = Describe("Podman pod create", func() {
 		session = podmanTest.Podman([]string{"run", fedoraMinimal, "curl", "-f", "localhost"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).To(ExitWithError())
+
+		session = podmanTest.Podman([]string{"pod", "create", "--network", "host"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "-dt", "--pod", session.OutputToString(), ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"inspect", "--format", "'{{.NetworkSettings.SandboxKey}}'", session.OutputToString()})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		Expect(session.OutputToString()).Should(ContainSubstring("''")) // no network path... host
```

Comment:
Please do not rely on podman inspect for such a test. podman inspect can have bugs too. I recommend to actually check the /proc/self/ns/net values.

Reply:
@Luap99 what do you mean my this? should I just `stat -c %d /proc/self/ns/net` and compare the two outputs? only thing is infra doesn't have stat and using any other image will mess this up.